pkg/flavor/kubernetes/nfs.go

```diff
@@ -904,7 +904,8 @@ func (flavor *Flavor) deleteNFSDeployment(name string, nfsNamespace string) erro
 	defer log.Tracef("<<<<< deleteNFSDeployment")
 
 	propagation := meta_v1.DeletePropagationBackground
-	options := &meta_v1.DeleteOptions{PropagationPolicy: &propagation}
+	gracePeriod := int64(5)
```

Comment:
nit: you can directly pass int64(5) in the DeleteOptions

Reply:
@raunakkumar we need to pass this as pointer hence separate variable.